assets/src/design-system/components/tabGroup/index.js

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { forwardRef, useRef } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+
+/**
+ * Internal dependencies
+ */
+import { Tab } from '../tab';
+
+const StyledTabGroup = styled.div`
+  display: flex;
+  align-items: flex-start;
+  flex-wrap: wrap;
+
+  & > button {
+    margin: 0 4px;
+    &:first-of-type {
+      margin-left: 0;
+    }
+    &:last-of-type {
+      margin-right: 0;
+    }
+  }
+`;
+
+export const TabGroup = forwardRef(function TabGroup(
+  {
+    getTabId = (id) => id,
+    getAriaControlsId,
+    tabs,
+    label,
+    handleTabClicked,
+    activeTabId,
+  },
+  ref
+) {
+  const tabRefs = useRef({});
+
+  return (
+    <StyledTabGroup ref={ref} aria-label={label} role="tablist">
+      {tabs.map(({ id, title, ...rest }) => {
+        const isActive = activeTabId === id;
+        return (
+          <Tab
+            key={uuidv4()}
```

Comment:
This will create a new `uuid` each render what we could do is generate one based on the title

```
import { fromString } from 'uuidv4';
...
fromString(title))
```

The issue with that becomes if we have two tabs in the same group with the same titles, but that feels like bad UX.

Reply:
i think that would be really odd if two tabs had the exact same title, i'll swap out for `fromString` - thanks for the rec!

Reply:
so i realized that i am working too hard here - I had only glanced at the new eslint rule set up last week about not allowing indexes and thought i'd read no ids either 🤦‍♀️ , so to maintain keys I've updated this to be `${title.replace(/\s/g, '')}_${id}` - no uuid necessary. 

I did look at using fromString but I learned that `uuidv4` the[ package is getting deprecated](https://www.npmjs.com/package/uuidv4) in favor of uuid (which is what's in use already) and anyway, while trying to figure out if fromString would still work I realized this doesn't need to be so hard. Sorry about that!